mk/spksrc.service.mk

```diff
@@ -229,6 +238,12 @@ ifeq ($(strip $(USE_DATA_SHARE_WORKER)),yes)
 		'."data-share" = {"shares": [{"name": $$share, "permission":{"rw":[$$user]}} ] }' $@ | sponge $@
 endif
 endif
+ifneq ($(strip $(SERVICE_CERT)),)
+	@jq --arg service "${SERVICE_CERT}" \
+	    --arg reload "${SERVICE_CERT_RELOAD}" \
+	    --arg display_name "${DISPLAY_NAME}" \
+		'."certificate-config" = {"services": [{"display_name": $$display_name, "display_name_i18n": $$display_name, "service":$$service} ], "reloader-relpath": $$reload } | ."service-cfg" = {}' $@ | sponge $@
+endif
```

Comment:
In that case, this just needs be excluded from DSM7.

Reply:
Done in latest commit.

Reply:
Thanks

Just an FYI the /usr/local paths are not used anymore since DSM7  (DSM6.2 also uses some of it) as per the HFS: https://github.com/SynoCommunity/spksrc/wiki/DSM-7.0 (table at the bottom)

Reply:
So is that compatible with DSM 6 as well?  On my DS418j running latest DSM 6.2.something, the `/var/packages/syncthing/var` link does not exist.  The one under `target/var` does however.

What should I use for maximum compatibility in the future?  Note that the `ca_reloader.sh` script does not receive any of the `SYNOPKG_PKG*` variables unfortunately.  At least under DSM 6.

Reply:
@publicarray I just added a check that will dynamically switch from `/var/packages/syncthing/var` to `/var/packages/syncthing/target/var` when necessary.  The former does not exist on my DSM 6.2 installation, and I've read several reports now that the latter does not work properly under DSM 7.